src/renderer/types/claim.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+// Actual claim type has more values than this
+// Add them as they are used
+export type Claim = {
```

Comment:
Should `Subscription` also be a standalone type file like `Claim`?

Reply:
Yes